module-3/web-app/package.json

```diff
@@ -52,6 +52,7 @@
   },
   "devDependencies": {
     "amazon-cognito-identity-js": "^5.2.9",
+    "axios": "^0.27.2",
```

Comment:
Instead of axios use native fetch api as we did for prosemirror in this draft pr https://github.com/USergei/bootcamp/blob/9b9cf016678c528a2635be3c7d8053f0b9720024/module-3/web-app/src/services/documentService.js

It's better to have as little external dependencies as possible

Reply:
Done